packages/core/utils/src/dml/modifiers/nullable.ts

```diff
@@ -1,6 +1,9 @@
-import { SchemaType } from "../types"
+import { MaybeFieldMetadata } from "../types"
 import { OptionalModifier } from "./optional"
 
+/**
+ * Nullable modifier marks a schema node as nullable
```

Comment:
suggestion: currently, for all nullable field our strategy was to make it optional and default to null by auto assigning, but I believe we should backport that

Reply:
Let's discuss over the call and then handle it in next PR :)